robot/impl/robot_reconfigure_test.go

```diff
@@ -2638,46 +2664,28 @@ func TestRemoteRobotsGold(t *testing.T) {
 			sensors.Named(resource.DefaultServiceName),
 			datamanager.Named(resource.DefaultServiceName),
 			arm.Named("arm1"),
-			arm.Named("foo:pieceArm"),
-			audioinput.Named("foo:mic1"),
-			camera.Named("foo:cameraOver"),
-			movementsensor.Named("foo:movement_sensor1"),
-			movementsensor.Named("foo:movement_sensor2"),
-			gripper.Named("foo:pieceGripper"),
+			arm.Named("foo:remoteArm"),
 			motion.Named("foo:builtin"),
 			sensors.Named("foo:builtin"),
 			datamanager.Named("foo:builtin"),
 		),
 	)
+
+	// start remote2's web service
 	err = remote2.StartWeb(ctx, options)
 	test.That(t, err, test.ShouldBeNil)
 
-	rr, ok := r.(*localRobot)
-	test.That(t, ok, test.ShouldBeTrue)
-
-	rr.triggerConfig <- struct{}{}
-
 	expectedSet := rdktestutils.NewResourceNameSet(
```

Comment:
If we're going to use a variable for this, I'd recommend naming it something like `mainPartAndFooAndBarResources`. The "expectedSet" is different depending on where we are in the test.